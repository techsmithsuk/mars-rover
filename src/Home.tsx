import React from 'react';
import './home.scss';

export function HomePage() {


    return (
        <div className="homePage">
            <section className="home" > Welcome to the Mars Rover!</section>
            <div className="sections">
                <section className="missions">Missions</section>
                <section className="weather">Weather</section>
                <section className="photos">Photos</section>
            </div>
        </div>
    );
}