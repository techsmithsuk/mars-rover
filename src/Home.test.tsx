import React from 'react';
import renderer from 'react-test-renderer';
import { HomePage } from './Home';

describe("Home Component", () => {
    it("Should render without errors", () => {
        const component = renderer.create(
            <HomePage/>
        );
        
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});