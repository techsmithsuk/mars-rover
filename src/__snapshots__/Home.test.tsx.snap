// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Should render without errors 1`] = `
<div
  className="homePage"
>
  <section
    className="home"
  >
     Welcome to the Mars Rover!
  </section>
  <div
    className="sections"
  >
    <section
      className="missions"
    >
      Missions
    </section>
    <section
      className="weather"
    >
      Weather
    </section>
    <section
      className="photos"
    >
      Photos
    </section>
  </div>
</div>
`;
